hack/testing/util.sh

```diff
@@ -177,7 +177,7 @@ function wait_for_fluentd_to_catch_up() {
     local uuid_es=$( uuidgen )
     local uuid_es_ops=$( uuidgen )
     local expected=${3:-1}
-    local timeout=${TIMEOUT:-300}
+    local timeout=${TIMEOUT:-600}
```

Comment:
ok - but I can't imagine what's going on that makes fluentd take 10 minutes to read a record from journal/json-file and store it in Elasticsearch :-P